src/setupTests.js

```diff
@@ -34,6 +37,8 @@ window.URL.createObjectURL = jest
 window.URL.revokeObjectURL = jest.fn()
 window.scrollTo = jest.fn()
 
+Element.prototype.scrollTo = jest.fn()
```

Comment:
A test for SynapseForm randomly started failing because this isn't implemented in JSDOM

Reply:
Strange